docs/api/saved-objects/get.asciidoc

```diff
@@ -50,3 +56,60 @@ The API returns the following:
   }
 }
 --------------------------------------------------
+
+The following example retrieves a dashboard object in the `testspace` by id.
+
+[source,js]
+--------------------------------------------------
+GET /s/testspace/api/saved_objects/dashboard/7adfa750-4c81-11e8-b3d7-01146121b73d
+--------------------------------------------------
+// KIBANA
+
+A successful call returns a response code of `200` and a response body
```

Comment:
Remove `A successful call returns a response code of `200` and a response body
containing a JSON structure similar to the following example:`, then replace with:

`The API returns the following:`